src/seedu/addressbook/AddressBook.java

```diff
@@ -306,13 +311,49 @@ private static void setupDefaultFileForStorage() {
 
     /**
      * Returns true if the given file is acceptable.
-     * The file path is acceptable if it ends in '.txt'
+     * The file path is acceptable if it does not contain any reserved string,
```

Comment:
Hmm, I meant to type reserved characters like <>:"/\|?*.
I will look towards how to better explain the behaviour.